src/components/Layout/Logo/index.js

```diff
@@ -0,0 +1 @@
+export * from './Logo';
```

Comment:
What's the intention of this `index.js`?

Reply:
It's in order to control/manipulate the API of importing this into a different module

```js
import { Layout, Component1, Component2 } from '../../components';

// Instead of

import { Layout } from '../components/Layout/Layout';

```